build.xml

```diff
@@ -123,18 +123,36 @@
 
         <exec command="composer install" dir="${drush.root}/.." passthru="true" />
 
+        <!-- Swap out the composer file for our git repository -->
+        <delete dir="${drush.root}/sites/all/modules/contrib/search_api_federated_solr" />
+        <symlink link="${drush.root}/sites/all/modules/contrib/search_api_federated_solr" target="../../../../../../../src/search_api_federated_solr-7.x" />
+
         <drush command="site-install" assume="yes">
             <option name="site-name">Drupal 7 - Federated Search</option>
             <option name="account-pass">admin</option>
             <param>standard</param>
         </drush>
+
+        <!-- The public files directory, and everything in it, needs to be world writable. -->
+        <exec command="chmod -R 777 ${drush.root}/sites/default/files" checkreturn="true" />
+
+        <!-- Copy demo files
+        <copy todir="${drush.root}/sites/default/files/field/image" mode="777">
+          <fileset dir="${drush.root}/sites/all/modules/federated_content/assets">
+            <include name="**/*.jpg" />
+          </fileset>
+        </copy> -->
+
+        <!-- Install default configuration and content. -->
         <drush command="pm-enable" assume="yes">
             <param>features_search_api_config</param>
-            <param>federated_content</param>
+            <param>federated_default_content</param>
         </drush>
-        <!-- The feature is not loading properly, but does reload correctly. -->
+        <!-- The public files directory, and everything in it, needs to be world writable. -->
+        <exec command="chmod -R 777 ${drush.root}/sites/default/files" checkreturn="true" />
```

Comment:
Fun fact: When drupal tries to write the first of these files, it makes the directory unwritable for the next set.

Reply:
Ugh, I know, it's frustrating! And it happens on all of our boxes IIRC. I honestly am not sure how to solve this, as everything that is within the Vagrant nfs share is owned by `dialout`.